app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -297,7 +298,17 @@ export default {
   mounted() {
     this.createEditorView();
     this.editorView.updateState(this.state);
-    this.focusEditor(this.value);
+    this.focusEditorInputField();
+
+    // BUS Event to insert text or markdown into the editor at the
+    // current cursor position.
+    // Components using this
+    // 1. SearchPopover.vue
```

Comment:
Didn't understand this statement?

Reply:
I added that comment so we could track the track the components using this bus event. Let me know if this is an overkill.